blog/BCHN-Financial-Report-2020-10-20.md

```diff
@@ -0,0 +1,218 @@
+        2020/09/27  ex:pr:video                  2.00000000 BCH
+        2020/09/30  ex:de:website:general        2.64317181 BCH
+        2020/09/30  ex:pr:representative         5.53097345 BCH
+        2020/10/01  ex:development:general      40.00000000 BCH
+        2020/10/02  ex:de:bounty:feature         0.50000000 BCH
+        2020/10/07  ex:development:bounty:test   1.25000000 BCH
+        2020/10/09  ex:development:bounty:doc    0.25000000 BCH
+        2020/10/10  ex:infrastructure            6.33552965 BCH
+
+In the time since the last report, BCHN activity was focused on general development (with backporting activity reduced due to developer resource constraints).
+Several bounties were awarded in this period for website-related content imports and test and feature developments.
+
+19.86275322 BCH were spent on website maintenance and improvement.
+
+54.35714286 BCH were spent on client software (and documentation) development.
+
+7.53097345 BCH were spent on public relations, including a video to introduce the team.
+
+6.33552965 BCH were spent on project infrastructure.
+
+
+### Basic statistics about BCHN project ledger
+
+These are derived from hledger tool's statistic report.
+
+        Main file                : bchn.journal
+        Included files           : prnc2exht3zxlrqqcat690tc85cvfuypngh7szx6mk.journal
+                                   pp03e95qz7yene7v4vdx2vfsqx4qsq4wfvw8z3pkjf.journal
+        Transactions span        : 2020-02-22 to 2020-10-14 (235 days)
+        Last transaction         : 2020-10-13 (7 days ago)
+        Transactions             : 207 (0.9 per day)
+        Transactions last 30 days: 17 (0.6 per day)
+        Transactions last 7 days : 1 (0.1 per day)
+        Payees/descriptions      : 1
+        Accounts                 : 20 (depth 4)
+        Commodities              : 2 (, BCH)
+
+
+### Notes about balances displayed on block explorers
+
+Block explorers may show a slightly lower amount for the main wallet address `prnc2exht3zxlrqqcat690tc85cvfuypngh7szx6mk` (2,109.93763055 BCH at block 657,989).
+
+This is because a small amount in donations (currently 0.00075823 BCH) are stored on a subsequent address (`pp03e95qz7yene7v4vdx2vfsqx4qsq4wfvw8z3pkjf`) in BCHN's multi-signature wallet which was issued to Coin.dance who are collecting donations on behalf of node client projects. They requested this separate address for their own accounting purposes. BCHN will probably periodically receive some donations coming in from there - should significant amounts accrue, they will eventually be moved to the main wallet, but the report (as above) will reflect the total donated amounts on both our main and this secondary address.
+
+
+### SLP donation address
+
+The project has a SimpleLedger (SLP) token donation address.
+
+`simpleledger:qrjv4klcn9z5kqeye3ps64pr6aa4mzux8cy776pm6k`
+
+Currently, the project has received and is holding donations of 2,000,000 SPICE 🌶️ and 1,000 USDt on that address.
```

Comment:
should this be `USDT` with capital `T`?

Reply:
It's officially `USDt` ([source](https://tether.to)), but most exchanges use `USDT`.